README.md

```diff
@@ -180,4 +180,4 @@ The most important root project Gradle tasks are listed in the table below.
 
 Copyright (C) 2017-2018 HERE Europe B.V.
 
-See the [LICENSE](../LICENSE) file in the root of this project for license details.
+See the [LICENSE](LICENSE) file in the root of this project for license details.
```

Comment:
Thanks for this fix! Mind writing the path as `./LICENSE` for clarity and consistency with similar code in this file?